### PR TITLE
refactor: Making use of new numeric typed-binary schemas.

### PR DESCRIPTION
**Pull request by iwoplaza**

### CODE REVIEW

File: packages/typegpu/tests/vector.test.ts [hunk 1]
```diff
@@ -60,7 +64,71 @@ describe('vec2f', () => {
...
+  it('should create a 2d vector with the given elements', () => {
+    const zero = d.vec2i(1, 2);
+    expect(zero.x).toEqual(1);
+    expect(zero.y).toEqual(2);
+  });
+
+  it('should create a 2d vector from the given scalar element', () => {
+    const zero = d.vec2i(5);
+    expect(zero.x).toEqual(5);
+    expect(zero.y).toEqual(5);
```
Comment (mhawryluk) [inline packages/typegpu/tests/vector.test.ts]: zero -> vec

File: packages/typegpu/tests/vector.test.ts [hunk 2]
```diff
@@ -121,7 +189,76 @@ describe('vec3f', () => {
...
+    expect(zero.y).toEqual(2);
+    expect(zero.z).toEqual(3);
+  });
+
+  it('should create a 3d vector from the given scalar element', () => {
+    const zero = d.vec3i(5);
+    expect(zero.x).toEqual(5);
+    expect(zero.y).toEqual(5);
+    expect(zero.z).toEqual(5);
+  });
```
Comment (mhawryluk) [inline packages/typegpu/tests/vector.test.ts]: zero -> vec

**Review by mhawryluk — APPROVED** [review]
Great! 😎

**Review by reczkok — APPROVED** [review]
Great change💜

**Pull request merged**